task01/script.js

```diff
@@ -0,0 +1,27 @@
+/*
+Timothy Washington
+script.js
+INFO 2134 - Javascript II
+Thoendel
+Creation date: Apr 12, 2020
+*/
+window.addEventListener('load', function() {
+    this.document.getElementById('myForm').addEventListener('submit',function(e){
+        let err='';//set up error hold
+        const regex=/(?:\(?[2-9]\d{2})(?:( ?\) ?)|( ?- ?))?(?:\d{3})(?: ?- ?)?(?:\d{4})/g;//regex
```

Comment:
This works well! The only scenario I found that ideally wouldn't pass is when you have only one parentheses.
Nicely done! 👍